### PR TITLE
Remove fork, update spotless, and smoke test framework improvements

### PR DESCRIPTION
**Pull request by trask**
Built on top of #2279, will rebase once that is merged

### CODE REVIEW

File: .pipelines/pipeline.user.windows.official.nonrelease.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ version:
 build:
   commands:
     - !!buildcommand
+      # TODO (heya) why do we need to upload the build scripts?
```
Comment (trask) [inline .pipelines/pipeline.user.windows.official.nonrelease.yml:9]:
```suggestion
```

File: .pipelines/pipeline.user.windows.official.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ version:
 build:
   commands:
     - !!buildcommand
+      # TODO (heya) why do we need to upload the build scripts?
```
Comment (trask) [inline .pipelines/pipeline.user.windows.official.yml:9]:
```suggestion
```

File: .pipelines/pipeline.user.windows.yml [hunk 3]
```diff
@@ -10,6 +10,7 @@ environment:
 versioning:
   commands:
     - !!defaultcommand
+      # TODO (heya) what is this used for?
```
Comment (trask) [inline .pipelines/pipeline.user.windows.yml:13]: we're not sure, TODO try remove?

File: .pipelines/pipeline.user.windows.yml [hunk 4]
```diff
@@ -28,6 +29,7 @@ static_analysis_options:
 restore:
   commands:
     - !!defaultcommand
+      # TODO (trask) test if this is needed, or maybe only needed for official release?
```
Comment (trask) [inline .pipelines/pipeline.user.windows.yml:32]: maybe ETW? TODO try to remove and see if ETW build still works

File: buildSrc/build.gradle.kts [hunk 5]
```diff
@@ -16,7 +16,9 @@ spotless {
 repositories {
   mavenCentral()
   gradlePluginPortal()
-  mavenLocal()
+  maven {
+    url = uri("https://oss.sonatype.org/content/repositories/snapshots")
```
Comment (trask) [inline buildSrc/build.gradle.kts:20]:
```suggestion
    // TODO (trask) remove this before making a release
    url = uri("https://oss.sonatype.org/content/repositories/snapshots")
```

**Review by heyams — APPROVED** [review]

**Pull request merged**